src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,7 @@
+import './Footer.scss';
+
+const Footer = (props): JSX.Element => <footer className="mwFooter">{props.children}</footer>;
+
+Footer.propTypes = {};
```

Comment:
Sometimes there is a `propTypes` assignment, sometimes there isn't. Could you please make sure propTypes is used everywhere?

Reply:
Yes will check and remove where it is not used